packages/protobuf-test/package.json

```diff
@@ -8,8 +8,8 @@
     "build:copy-gen-js": "rsync -a --exclude '*.js' src/gen/js dist/types/gen && rsync -a --exclude '*.d.ts' src/gen/js dist/esm/gen",
     "pregenerate": "rm -rf src/gen/*/* descriptorset.*",
     "generate": "npm run generate:ts && npm run generate:js",
-    "generate:ts": "protoc --experimental_editions --es_out=src/gen/ts --es_opt=ts_nocheck=false,target=ts     --proto_path=. $(buf ls-files extra) --proto_path=$(upstream-include test) $(upstream-files test) google/protobuf/type.proto",
-    "generate:js": "protoc --experimental_editions --es_out=src/gen/js --es_opt=ts_nocheck=false,target=js+dts --proto_path=. $(buf ls-files extra) --proto_path=$(upstream-include test) $(upstream-files test) google/protobuf/type.proto",
+    "generate:ts": "protoc --es_out=src/gen/ts --es_opt=ts_nocheck=false,target=ts     --proto_path=. $(buf ls-files extra) --proto_path=$(upstream-include test) $(upstream-files test) google/protobuf/type.proto",
```

Comment:
Editions are no longer experimental, and we can remove the flag `--experimental_editions` here (and at several other places).